docs/content/docs/ops/metrics.md

```diff
@@ -1343,6 +1343,11 @@ Note that for failed checkpoints, metrics are updated on a best efforts basis an
       <td>The time in nanoseconds that elapsed between the creation of the last checkpoint and the time when the checkpointing process has started by this Task. This delay shows how long it takes for the first checkpoint barrier to reach the task. A high value indicates back-pressure. If only a specific task has a long start delay, the most likely reason is data skew.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td>checkpointRestoreTime</td>
+      <td>The time in milliseconds that one task spends on restoring/initialization, return 0 when the task is not in initialization/running status.</td>
+      <td>Counter</td>
```

Comment:
How about calling it `initializationTime` instead of `checkpointRestoreTime`? 
Currently, it includes something besides of restoring from checkpoint; and there might be more initialization in the future.
In the docs, we could say that most of the initialization time is usually spent in restoring from the checkpoint.

The section would need to be changed then, maybe to `Availability`.

WDYT?

Reply:
I renamed `checkpointRestoreTime` to `initializationTime` in subtask level, and moveed it to [`IO` section](https://nightlies.apache.org/flink/flink-docs-master/docs/ops/metrics/#io).
For `Availability` section, I think it's a metric that describes job level status, we can reuse `initializingTime/initializingTotalTime`.